src/index.ts

```diff
@@ -203,38 +170,34 @@ const weightedchance_message = async (
   sender: Chatter
 ) => {
   if (chance == -1) {
-    return (
-      sender + ", looks like you're not in the queue. Try !add XXX-XXX-XXX."
-    );
+    return i18next.t("submitterNotFound", { submitter: sender });
   } else if (chance == -2) {
-    return (
-      sender +
-      ", you are in a BRB state, so you cannot be selected in weighted random. Try using !back and then checking again."
-    );
+    return i18next.t("oddsBRB", { sender });
   } else if (chance === 0) {
-    return "Your level is being played right now!";
+    return i18next.t("positionCurrent");
+  }
+
+  if (multiplier > 1.0) {
+    return i18next.t("senderOddsMultiplier", {
+      sender,
+      chance,
+      multiplier: multiplier.toFixed(1),
```

Comment:
Here is another usage of `toFixed` which would need to be changed.
Numbers could be formatted like this: https://www.i18next.com/translation-function/formatting#passing-options-to-the-formatting using the `minimumFractionDigits`.